src/js/editor/event-manager.js

```diff
@@ -45,6 +45,14 @@ export default class EventManager {
     this._textInputHandler.register(inputHandler);
   }
 
+  unregisterInputHandler(name) {
+    this._textInputHandler.unregister(name);
+  }
+
+  unregisterAllTextInputHandlers() {
+    this._textInputHandler.destroy();
```

Comment:
This needs to create a new text input handler afterward, right? So that someone can register new handlers after the unregister, like this:
```
editor.unregisterAllTextInputHandlers();
editor.onTextInput({...}); // there needs to be a non-destroyed input handler
```